APE0.rst

```diff
@@ -157,8 +157,8 @@ In the case of a vacancy partway through a term, a by-election will be held.
 The term of the newly-elected member runs for the remainder of the term of
 the member they are replacing. 
 
-For the initial election of Coordination Committee members, two seats will have
-terms of one year, one has a term of two years, and one has a term of three
+For the initial election of Coordination Committee members (four at the time instead of five), two seats will have
+terms of one year, one has a term of two years (two here now that there are five), and one has a term of three
```

Comment:
I would find it useful if the section of the APE that describes the terms of the first CoCo would describe the terms of the actual first CoCo instead of some draft proposal:
```suggestion
For the initial election of Coordination Committee members, two seats will have
terms of one year, two have a term of two years, and one has a term of three
```
However, I am not a lawyer so I cannot claim with full confidence that such a change would be allowed.